sources/github.go

```diff
@@ -25,8 +25,9 @@ import (
 )
 
 var (
-	_        Source = &githubSource{}
-	githubCI        = "/actions"
+	_                Source = &githubSource{}
+	githubCI                = "/actions"
+	workflowFileName        = "build-release-policy.yaml"
```

Comment:
This will create a dependency on the file name, leading to unexpected behaviour/failures in the future. Is there any API that can list the workflows of a specific repo? take the failed workflow id and use https://github.com/google/go-github/blob/d4c1caf4910c4a29f7bf3b858b9985083ff42683/github/actions_workflows.go#L155